config/v2_3_experimental/types/unit.go

```diff
@@ -30,14 +30,35 @@ var (
 	ErrInvalidNetworkdExt = errors.New("invalid networkd unit extension")
 )
 
+func NewNoInstallSectionError(name string) error {
```

Comment:
Imo this should go in `config/errors` and have every config's `types/unit.go` import that.

Reply:
Sure; it looked like there were a bunch of duplicated errors between the config versions, so I assumed there was some weird reason to copy-paste all this code.

I had another commit to add a 'shared' pkg and move stuff into there which I wanted to propose as a better way to handle this duplication, but I figured that would be best as a separate PR.

Now that I know config/errors exists, I can move it there though, thanks!

Reply:
`config/errors` was added as a common place for this stuff in that big refactor I did, but I really just moved stuff that needed to be moved. It would probably make sense as a followup to find and move all the rest of the errors over.